src/main/scala/akka/contrib/datareplication/ORMultiMap.scala

```diff
@@ -66,7 +66,7 @@ case class ORMultiMap private (private[akka] val map: ORMap)
   /**
    * Convenience for put. Requires an implicit Cluster.
    */
-  def +(entry: (String, Set[Any]))(implicit node: Cluster): ORMultiMap = {
+  def +[T](entry: (String, Set[T]))(implicit node: Cluster): ORMultiMap = {
```

Comment:
yes, I found that this was not possible with `Any`: 
`ORMultiMap().put(address2, "b", Set("B1", "B2", "B3"))`